public/app/plugins/datasource/elasticsearch/query_builder.ts

```diff
@@ -55,6 +55,10 @@ export class ElasticQueryBuilder {
 
     if (aggDef.settings.min_doc_count !== void 0) {
       queryNode.terms.min_doc_count = parseInt(aggDef.settings.min_doc_count, 10);
+
+      if (isNaN(queryNode.terms.min_doc_count)) {
```

Comment:
This feels wrong, but wanted to keep backward compatibility that uses `parseInt`. Any better ideas?

Reply:
One thing that we should think about is sanitizing the input text, wdyt?

Reply:
Think this is a good idea

Reply:
don't know that much about `ngSanitize` and it's Angular but I've seen us use the `sanitize` function in a lot of places lately so I'd go with that one.